src/components/Volunteers/CardVolunteerFull.jsx

```diff
@@ -1,128 +1,156 @@
       >
-        Deny
+        Has W9
       </Button>
       <Button
-        onClick={() => props.refer._approveAmbassador(props.volunteer, true)}
+        onClick={() => props.refer._updateW9Ambassador(props.volunteer, false)}
       >
-        Approve
+        Does Not Have W9
       </Button>
-    </div>
-    Currently Admin: {props.volunteer.admin ? "Yes" : "No"}
-    <div>
-      <Button onClick={() => props.refer._adminizeAmbassador(props.volunteer)}>
-        Make Admin
+      <div>
+        PayPal Approved?: {props.volunteer.paypal_approved ? "Yes" : "No"}
+      </div>
+      <Button
+        onClick={() =>
+          props.refer._updatePayPalApprovedAmbassador(props.volunteer, true)
+        }
+      >
+        PayPal Approved
       </Button>
-    </div>
-    <h2>Profile</h2>
-    <VolunteerProfile volunteer={props.volunteer} refer={props.refer} />
-    <h2>Verification</h2>
-    <div>
-      {props.volunteer.verification ? (
-        <div>{JSON.stringify(props.volunteer.verification, null, 2)}</div>
-      ) : (
-        <div>no phone lookup</div>
-      )}
-    </div>
-    <h2>Accounts</h2>
-    <div>
-      {props.volunteer.account ? (
-        <div>{JSON.stringify(props.volunteer.account, null, 2)}</div>
-      ) : (
-        <div>no accounts</div>
-      )}
-    </div>
-    <h2>Payouts</h2>
-    <div>
-      {props.volunteer.payouts ? (
-        <div>{JSON.stringify(props.volunteer.payouts, null, 2)}</div>
-      ) : (
-        <div>no payouts</div>
-      )}
-    </div>
-    <h2>Quiz Results</h2>
-    <div>
-      {props.volunteer.quiz_results ? (
-        props.volunteer.quiz_results.map ? (
-          props.volunteer.quiz_results.map((qr) => {
-            return (
-              <div key={qr.question}>
-                <b>question:</b> {qr.question} <b>answer:</b> {qr.answer}
-                <br />
-                <br />
-              </div>
-            );
-          })
+      <Button
+        onClick={() =>
+          props.refer._updatePayPalApprovedAmbassador(props.volunteer, false)
+        }
+      >
+        Not PayPal Approved
+      </Button>
+      <div>Currently Approved? {props.volunteer.approved ? "Yes" : "No"}</div>
+      <div>
+        <i>
+          Pressing Approve will only change approval status, not locked status.
+        </i>
+      </div>
+      <div>
+        <Button
+          onClick={() => props.refer._approveAmbassador(props.volunteer, false)}
+        >
+          Deny
+        </Button>
+        <Button
+          onClick={() => props.refer._approveAmbassador(props.volunteer, true)}
+        >
+          Approve
+        </Button>
+      </div>
+      Currently Admin: {props.volunteer.admin ? "Yes" : "No"}
+      <div>
+        <Button
+          onClick={() => props.refer._adminizeAmbassador(props.volunteer)}
+        >
+          Make Admin
+        </Button>
+      </div>
+      <div>
+        Claim Tripler Limit:{" "}
+        <Input
+          type="number"
```

Comment:
Instead of `type="number"`, use `inputmode="numeric"`

This is a good explanation why: https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/

And this is a survey of all the different `inputmode` options:
https://css-tricks.com/everything-you-ever-wanted-to-know-about-inputmode/

Reply:
oh, thanks, I didn't know about that!